src/main/Config.h

```diff
@@ -32,8 +32,17 @@ class Config : public std::enable_shared_from_this<Config>
     {
         VALIDATOR_LOW_QUALITY = 0,
         VALIDATOR_MED_QUALITY = 1,
-        VALIDATOR_HIGH_QUALITY = 2
+        VALIDATOR_HIGH_QUALITY = 2,
```

Comment:
forgot to update documentation in `docs/stellar-core_example.cfg` (where `QUALITY` is mentioned)

Reply:
forgot to update `kQualities` (so `CRITICAL` cannot be parsed at all)